### ISSUE TITLE
Update outdated design docs

### ISSUE THREAD

**Issue by JohanIIS** [dotse/zonemaster#573]
Motion to update, move, remove or extract vital information from the following files:

* [/docs/design/Architecture Notes.md](https://github.com/dotse/zonemaster/blob/master/docs/design/Architecture%20Notes.md)
* [/docs/design/Implementation Guidelines.md](https://github.com/dotse/zonemaster/blob/master/docs/design/Implementation%20Guidelines.md)

They contain good information but have been left unfinished since 2013/2014 (one file even ends with "to be continued"). They should either be resurrected and updated, moved to an archive folder, removed entirely or have the information within saved in another document.

**Comment by sandoche2k** [comment]
@JohanIIS Should it be part of the public github?

**Comment by mattias-p** [comment]
I suggest that the files are left where they are, but that a clear note about the documents being out of date is added to the top of the documents.

**Comment by matsduf** [comment]
I disagree. This issue is to do something meaningful with the documents. In the mean time we could put a note that they need update, but this issue should not be closed until we have updated documents or used the relevant information in the documents in other documents and deleted these documents.

**Comment by sandoche2k** [comment]
+1

**Comment by matsduf** [comment]
F2F: update or remove.

**Comment by matsduf** [comment]
#1087 proposes to remove the old documents.
